python/python/bystro/tada_mixture/mixture_tada_em.py

```diff
@@ -0,0 +1,394 @@
+"""
+This uses the expectation maximization algorithm to fit a latent variable
+model for Tada, with two flavors, a ZIP and a standard poisson. The 
```

Comment:
somewhere (maybe not here but somewhere) we should have a link to the papers we're implementing / extending